src/store/tests/__tests__/tests.analytics.effects.spec.ts

```diff
@@ -210,26 +210,4 @@ describe('Tests Analytics Effects', () => {
       });
     });
   });
-
-  describe('setActivityCode', () => {
```

Comment:
Could we move this to non pass analytics effect spec as this is almost identical and therefore the test should be almost plug and play?